src/main/org/nlogo/swing/ModalProgressTask.scala

```diff
@@ -63,8 +63,8 @@ object ModalProgressTask {
   extends Thread("ModalProgressTask#Boss") {
     override def run() {
       try {
-        while (!dialog.isVisible)
-        Thread.sleep(50)
+        while (!dialog.isVisible) Thread.sleep(50)
```

Comment:
I would prefer if this was styled
```scala
while (!dialog.isVisible) {
  Thread.sleep(50)
}
//...
```
I would be fine with omitting the braces, but `Thread.sleep(50)` needs its own line.